### PR TITLE
Update boot2docker script to use API

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

**Comment by cjellick** [comment]
@hibooboo2 I updated the register-boot2docker script to use the API to get the "proper" `docker run` command. If you have the time/inclination, do you want to try this out and see if it works for you as well?

If you don't/didn't use register-boot2docker.sh as part of your workflow, don't worry about trying it out.

**Comment by cjellick** [comment]
...It now requires jq. Not sure if that's a deal breaker.

File: tools/development/register-boot2docker.sh [hunk 1]
```diff
@@ -11,10 +10,14 @@ fi
 # This is just here to make sure your environment is sane
 docker info
 
-CONSOLE_ARGS=""
-if [ -t 1 ]; then
```
Comment (cloudnautique) [inline tools/development/register-boot2docker.sh]: These allow interrupting the process while its running. Something ibuildthecloud does... Doesn't look like they are being set/used

Comment (cjellick) [inline tools/development/register-boot2docker.sh]: Ah, was curious about why those were needed. Thanks for clearing up. I can add them back in.

**Comment by ibuildthecloud** [comment]
The curl commands don't need to be so ugly.  The api supports multi part forms so you can do, "curl -F key=value"

**Comment by cjellick** [comment]
Ping @ibuildthecloud and @cloudnautique.
Fixed it up. Can you guys take another look?

**Comment by cloudnautique** [comment]
We need Jq added to bootstrap. This script is called in ./scripts/ci

**Comment by cjellick** [comment]
I incorrectly assumed that this script was only used for registering boot2docker instances. Will probably need to refactor more.

**Pull request closed without merging**